JeffKersting/blockTracker
# Testing cypress setup

#### What does  this PR do?

Installs cypress test suite. Adds a fixture for coin intercepts. Tests login page functionality, including error handling.

#### Where should the reviewer start?

In the cypress folder, review `coins.json` added to fixtures folder, and `login-page.js` tests in integration folder.

#### Is this a bug fix or a feature?

Feature

#### How should this be manually tested?

In terminal, enter `npx cypress open` and ensure that all login-page tests are passing.
